### PR TITLE
Create simple tutorial for beacon-kit rollup

### PR DESCRIPTION
**Pull request by yarikbratashchuk**
## Summary by CodeRabbit

- **New Features**
  - Added new tutorial links for "BeaconKit Ethereum rollup" and "BeaconKit EVM" in the sidebar under the "Tutorials" section.
  - Introduced a comprehensive guide for developers on building and operating a sovereign rollup using BeaconKit and Rollkit, including detailed setup instructions, customization recommendations, and troubleshooting tips.

### CODE REVIEW

File: .vitepress/config.ts [hunk 1]
```diff
@@ -250,6 +250,7 @@ function sidebarHome() {
             { text: "CosmWasm rollup", link: "/tutorials/cosmwasm" },
             { text: "Omni Octane EVM rollup", link: "/tutorials/octane-evm" },
             { text: "Polaris EVM rollup", link: "/tutorials/polaris-evm" },
+	    { text: "BeaconKit Ethereum rollup", link: "/tutorials/beaconkit" },
```
Comment (Manav-Aggarwal) [inline .vitepress/config.ts:253]: nit: rename to BeaconKit EVM

File: tutorials/beaconkit.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+```bash
+cd $HOME
+git clone -b rollkit https://github.com/rollkit/beacon-kit.git
+cd beacon-kit && git checkout rollkit
+```
+
+## Start the Go Ethereum (Geth) client {#start-go-ethereum-client}
+
+Start a Go Ethereum client to provide the Ethereum layer for your rollup:
```
Comment (Manav-Aggarwal) [inline tutorials/beaconkit.md:40]: nit: modify to "provide the execution layer"

File: tutorials/beaconkit.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+```bash
+cd $HOME/beacon-kit
+make start-geth
+```
+
+This command initializes a new Ethereum node with a specified genesis file and starts the Geth node with specific configuration options.
```
Comment (Manav-Aggarwal) [inline tutorials/beaconkit.md:47]: Ethereum node? Should be renamed to Rollkit node right?

**Review by Manav-Aggarwal — CHANGES_REQUESTED** [review]

File: tutorials/beaconkit.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+## Introduction
+
+This tutorial guides developers through the process of building and running a sovereign rollup using BeaconKit and Rollkit. BeaconKit is a modular framework for building EVM-based consensus clients, while Rollkit is a modular framework for building sovereign rollups. By combining these tools, you can create your own customizable rollup chain using the Ethereum Virtual Machine (EVM).
+
+## Prerequisites
+
+Before starting, ensure you have the following software installed on your machine:
+
+* [Rollkit](https://rollkit.dev/): A modular framework for sovereign rollups
```
Comment (MSevey) [inline tutorials/beaconkit.md:13]: We should include the install instructions as well.

File: tutorials/beaconkit.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+* [Docker](https://www.docker.com/): For running containerized applications
+* [Go](https://golang.org/) (version >= 1.21.0): The programming language used by BeaconKit
```
Comment (MSevey) [inline tutorials/beaconkit.md:15]: Same

File: tutorials/beaconkit.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+* [jq](https://stedolan.github.io/jq/): A lightweight command-line JSON processor
```
Comment (MSevey) [inline tutorials/beaconkit.md:16]: Isn't this also in some install script?

Comment (yarikbratashchuk) [reply tutorials/beaconkit.md:16]: it is, but shouldn't we mention it anyway?

File: tutorials/beaconkit.md (hunk 3 above)
Comment (yarikbratashchuk) [reply tutorials/beaconkit.md:47]: ethereum node

File: tutorials/beaconkit.md (hunk 4 above)
Comment (yarikbratashchuk) [reply tutorials/beaconkit.md]: should we just point out to the tutorial where we cover installation

**Comment by jcstein** [comment]
PR 420, how fitting 😄

File: .vitepress/config.ts [hunk 7]
```diff
@@ -250,6 +250,7 @@ function sidebarHome() {
             { text: "CosmWasm rollup", link: "/tutorials/cosmwasm" },
             { text: "Omni Octane EVM rollup", link: "/tutorials/octane-evm" },
             { text: "Polaris EVM rollup", link: "/tutorials/polaris-evm" },
```
[the remark this answers is not in the document]
Comment (Manav-Aggarwal) [reply .vitepress/config.ts:252]: We can remove it in a follow up PR once we consolidate the front-end guides

**Review by Manav-Aggarwal — APPROVED** [review]

**Pull request merged**